deploy/060_servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: stuck-ebs-vols
```

Comment:
I would lock the namespace of the ServiceMonitor as you did for the other objects.
Considering it will be deployed by an automated cmd, if you don't lock it might be created in another ns